### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by wchesh24**
Added team name +

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1,2 @@
 print("helloworld")
+print("Junior Dev Struggle Bus")
```
Comment (AAspCodes) [inline main.py]: i dont like it

Comment (wchesh24) [reply main.py]: You are wrong

Comment (AAspCodes) [reply main.py]: im your boss

File: main.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 print("helloworld")
-print("Junior Dev Struggle Bus")
\ No newline at end of file
+print("Junior Dev Struggle Bus")print("Is this better")
```
Comment (AAspCodes) [inline main.py]: looks better

**Pull request merged**